lib/Makefile

```diff
@@ -38,3 +38,5 @@ all: bsc.exe bsb.exe bsppx.exe bsb_helper.exe  refmt.exe reactjs_jsx_ppx_2.exe
 # bsppx.exe is needed for merlin
 # reactjs_jsx_ppx_3.exe
 # .PHONY: test
```

Comment:
this diff is not needed

Reply:
the ninja.darwin should not be removed

Reply:
The makefile commands won't work properly on my machine without the specifiying phony. But yeah, this is related to the workflow just like how https://github.com/BuckleScript/bucklescript/issues/2691 is so a change like this shouldn't belong here

Reply:
Also, any what could have caused my ninja.darwin to be removed when I followed the setup instructions?